MyComics/MyComics.xcodeproj/xcuserdata/silvia.espana.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -116,29 +116,29 @@
             landmarkType = "3">
             <Locations>
                <Location
-                  uuid = "EE7629D3-E8DF-4B0C-99A8-8A646F6CA6F7 - 9ad1e66833dba1ef"
+                  uuid = "EE7629D3-E8DF-4B0C-99A8-8A646F6CA6F7 - a1a795c2a4f2d7db"
```

Comment:
Add missing lines on to gitignore file

### Xcode Patch ###
*.xcarchive
*.xcodeproj/*
!*.xcodeproj/project.pbxproj
!*.xcodeproj/xcshareddata/
!*.xcworkspace/contents.xcworkspacedata
/*.gcno

### Projects ###
*.xcodeproj
*.xcworkspace